src/eos-updater-util.h

```diff
@@ -37,21 +37,29 @@
 G_BEGIN_DECLS
 
 #define EOS_UPDATER_ERROR (eos_updater_error_quark())
-GQuark eos_updater_error_quark (void);
+GQuark
```

Comment:
It's normal for headers to have the return type on the same line in GObject style code.